script2.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Generates a random number for rgb
+ *
+ * @returns {number}
+ */
+let x = () => Math.floor(Math.random() * 256);
+
+for (let i = 1; i <= 10; i++) {
+
+    let randomRGB = `rgb( ${x()}, ${x()}, ${x()} )`;
+    let html = `<div style="background-color: ${randomRGB}">${i}</div>`;
+    document.querySelector('main').innerHTML += html
```

Comment:
I would probably make a function that builds the randomRGB string by itself and then add that to the html separately. Its a good practice to separate logic into modular chunks. Like lets say you wanted to generate that randomRGB and use it elsewhere - having it in a function will allow you to do that easier.